README.md

```diff
@@ -2,6 +2,36 @@
 
 Implementation of a https://polkadot.network node in Rust.
 
+## Using Docker
+
+### The easiest way
+
+The easiest/faster option is to use the latest image.
+
+Let´s first check the version we have. The first time you run this command, the polkadot docker image will be downloaded. This takes a bit of time and bandwidth, be patient:
+
+```
+docker run --rm -it polkadot:0.2.0 polkadot --version
+```
+
+You can also pass any argument/flag that polkadot supports:
+
+```
+docker run --rm -it polkadot:0.2.0 polkadot --name "PolkaDocker"
+```
+
+Once you are done experimenting and picking the best node name :) you can start polkadot as daemon, exposes the polkadot ports and mount a volume that will keep your blockchain data locally:
+
+```
+docker run -d -p 30333:30333 -p 9933:9933 -v /my/local/folder:/root/.local/share/Polkadot polkadot:0.2.0 polkadot
+```
+
+
+### Build your own image
+To get up and running with the smallest footprint on your system, you may use the Polkadot Docker image.
+You can either build it yourself (it takes a while...):
+
+
```

Comment:
some information missing here?

Reply:
Yes, hard to split the docker part. 
I agree the building part should probably come later.

I moved the whole docker to the end in my last commit, including the (indeed) missing part.

The docker part describing how to 'just run' polkadot may be however interesting for many as it allows starting a node with the least amount of effort and dependencies beside a valid docker install. This is also much much quicker than to build the whole project.